R/pkgdown.R

```diff
@@ -28,6 +28,11 @@ use_pkgdown <- function(config_file = "_pkgdown.yml", destdir = "docs") {
   use_build_ignore("pkgdown")
   use_git_ignore(destdir)
 
+  ui_todo(
```

Comment:
I wonder if this is actually sufficiently important we should force people to supply it?

Reply:
I think you often don't know it at this point? I wouldn't want it to be a hard blocker.